docs/reference/elasticsearch/index.md

```diff
@@ -1,18 +1,10 @@
 # Elasticsearch and index management
 
-% TO-DO: Add links to "Elasticsearch basics"%
-
-This section contains reference information for Elasticsearch and index management features, including:
-
-* Settings
-* Security roles and privileges
-* Index lifecycle actions
-* Mappings
-* Command line tools
-* Curator
-* Clients
-
-% TO-DO: Add links to "query language and scripting language sections"%
-
-Elasticsearch also provides REST APIs that are used by the UI components and can be called directly to configure and access Elasticsearch features.
-Refer to [Elasticsearch API](https://www.elastic.co/docs/api/doc/elasticsearch) and [Elasticsearch Serverless API](https://www.elastic.co/docs/api/doc/elasticsearch-serverless).
\ No newline at end of file
+This section contains reference information for Elasticsearch and index management features.
+ 
+Elasticsearch also provides the following REST APIs:
```

Comment:
This sentence doesn't lead into the list accurately, because the list contains more than just REST APIs.

Reply:
@leemthompo Thanks for spotting this!
I slightly adjusted both text and links. Can you check if it's OK?

Reply:
text looks good now 👍 looks like got link errors tho